db/schema.rb

```diff
@@ -55,19 +68,19 @@
     t.datetime "updated_at", precision: 6, null: false
     t.string "repository_link"
     t.bigint "lectures_id"
-    t.integer "status", default: 0, null: false
+    t.integer "status"
     t.index ["lectures_id"], name: "index_course_sessions_on_lectures_id"
   end
 
   create_table "homework_submissions", force: :cascade do |t|
-    t.bigint "user_id", null: false
     t.bigint "homework_id", null: false
     t.string "pull_request"
     t.boolean "is_public"
     t.datetime "created_at", precision: 6, null: false
-    t.datetime "updated_at", precision: 6, null: false 
+    t.datetime "updated_at", precision: 6, null: false
+    t.bigint "course_session_participant"
```

Comment:
this too

Reply:
Removed

Reply:
it came back after migration. I think we need to clean this up somewhere.. I think lets get this PR in and handle it tomorrow